site/en/docs/webstore/manage/index.md

```diff
@@ -103,6 +103,62 @@ store item by adding `/reviews` at the end of your item’s URL:
 
 {% endAside %}
 
+## Provide user support
+
+To ensure the best user experience and build a great extension it's important to
+collect, evaluate, and follow-up on user feedback. You can manage your extension's user feedback in
```

Comment:
...and follow up on user feedback. (Not hyphenated in this case.)

Style note: "a follow-up call" would be hyphenated, because it's a compound adjective that precedes the noun it modifies. See also [Google style](https://developers.google.com/style/hyphens#compound-modifiers) and [Grammarly](https://www.grammarly.com/blog/hyphen/) etc. to continue down this rabbit hole...